vllm/model_executor/layers/quantization/gguf.py

```diff
@@ -355,7 +378,9 @@ class GGUFLinearMethod(LinearMethodBase):
         quant_config: The GGUF quantization config.
     """
 
-    def __init__(self, quant_config: GGUFConfig):
+    def __init__(self, quant_config: GGUFConfig, model_arch: str = ""):
+        self.model_arch = model_arch.lower()
+        self.is_gemma3 = "gemma3" in self.model_arch
```

Comment:
```suggestion
    def __init__(self, quant_config: GGUFConfig):
```
We shouldn't couple model_arch with quant_method.